tests/lms/services/canvas_api_test.py

```diff
@@ -14,8 +14,10 @@
 from lms.services.canvas_api import CanvasAPIClient
 
 
-class TestCanvasAPIClient:
-    def test_get_token_sends_an_access_token_request(
+class TestGetToken:
+    """Unit tests for CanvasAPIClient.get_token()."""
```

Comment:
I wonder if we ought to include the class name in the test class name as well to avoid needing these comments (eg. `TestCanvasAPIClient_GetToken`). That might be a bit redundant though if the module being tested only has one class.